cursesman/entities.py

```diff
@@ -92,8 +95,8 @@ def __init__(self, name, x, y, col=0):
         super().__init__(name, x, y, col=col)
         self.speed = 1
         # maybe not all characters will have the ability to drop bombs
-        self.max_bombs = 1
```

Comment:
i think you start with 1

Reply:
Ahh sorry, modified for debugging purposes